examples3/src/main/scala/sttp/client4/examples/WebSocketOx.scala

```diff
@@ -0,0 +1,32 @@
+package sttp.client4.examples
+
+import ox.*
+import ox.channels.Source
+import sttp.client4.*
```

Comment:
@adamw There is a slight inconvenience that I keep running into in such cases. Typically I import `sttp.client4.*` first. Then I add some more code, for example a method with `(using Ox)`. My IDE suggests importing `ox.Ox`, which goes to the end of import list. This becomes an error, because `value Ox is not a member of sttp.client4.ox`. Since I am aware of this, I quickly add `import ox.*` as the first import to fix the issues, but I can imagine that such errors may confuse people pretty frequently. 
This is not only a matter of ox, some other integrations use the same pattern (like `fs2`). Or maybe it's just a problem of Metals, which should be smarter and put the import before `sttp.client4.*` automatically? How does this work for you in IJ? :)

Reply:
yeah that's a problem; it's only solved for tapir+ZIO by having a `ztapir` package. So ideally we would need to have some other package name for sttp-ox and tapir-ox integrations than `xyz.ox` ;)

Reply:
Maybe we could add one level, like `sttp.client4.interop.ox`?

Reply:
I don't think this would solve the issue, since the problem is that the last component of the package is called `ox`, same as the top-level `ox` package?

Reply:
Ah no, maybe not. Let's try then `sttp.client4.integ.ox`, as in tapir-cats for example?

Reply:
Or maybe as in other sttp4 modules, we've got e.g. `sttp.client4.impl.cats` and `sttp.client4.httpclient.cats`. We could do the same here?

Reply:
So `sttp.client4.impl.ox`? Ok, this would be consisted. I was pretty sure we had `sttp.client4.cats` and `sttp.client4.fs2`, etc. but there's indeed `impl`, so let's follow this convention.

Reply:
Yeah, not the most beautiful, but might just work

Reply:
I would prefer `interop` to `impl`, but let's be consistent.